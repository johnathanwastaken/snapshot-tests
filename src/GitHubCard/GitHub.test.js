import Profile from "./GitHubCard"
import renderer from "react-test-renderer"

test('Testing for snapshot', () => {
    const tree = renderer.create(<Profile/>).toJSON()
    expect(tree).toMatchSnapshot()
})