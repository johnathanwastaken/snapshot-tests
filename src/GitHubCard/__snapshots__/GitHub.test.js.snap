// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing for snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
  >
    <img
      className="card-img-top"
      src="https://avatars.githubusercontent.com/u/97579743?v=4"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Johnathan Ortiz-Flores
      </div>
      <p
        className="card-text"
      >
        I need sleep.
      </p>
    </div>
  </div>
</div>
`;
